descriptors/ModuleDescriptor-template.json

```diff
@@ -529,6 +534,19 @@
         "inn-reach.patron-type-mappings.collection.put"
       ]
     },
+    {
+      "permissionName" : "inn-reach.contribution.item.post",
+      "displayName" : "start initial contribution process",
+      "description" : "Start initial contribution process"
+    },
+    {
+      "permissionName" : "inn-reach.contribution.all",
```

Comment:
lets align the naming with existing permissions

inn-reach.**contributions**.item.post
inn-reach.**contributions**.all